bundles/redhat8.6/bundle.sh.gotmpl

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -euo pipefail
+sed -i 's/\(def in_container():\)/\1\n    return False/g' /usr/lib64/python*/*-packages/rhsm/config.py
+FOUND=false
+
+RHSM_ORG_ID=${RHSM_ORG_ID:-""}
+RHSM_ACTIVATION_KEY=${RHSM_ACTIVATION_KEY:-""}
+RHSM_USER=${RHSM_USER:-""}
+RHSM_PASS=${RHSM_PASS:-""}
+
+if [[ -n "${RHSM_ORG_ID}" && -n "${RHSM_ACTIVATION_KEY}" ]]; then
+  subscription-manager register --org="${RHSM_ORG_ID}" --activationkey="${RHSM_ACTIVATION_KEY}" --force
+  FOUND=true
+fi
+
+if [[ -n "${RHSM_USER}" && -n "${RHSM_PASS}" && ${FOUND} == false ]]; then
+  subscription-manager register --username="${RHSM_USER}" --password="${RHSM_PASS}" --force
+  FOUND=true
+fi
+
+if [[ ${FOUND} == false ]]; then
+  echo "You must use subscription manager to fetch packages for redhat"
+  exit 1
+fi
+
+subscription::unregister() {
+  subscription-manager unregister
+}
+
+subscription::defer_unregister() {
+    trap subscription::unregister ABRT
+    trap subscription::unregister EXIT
+    trap subscription::unregister HUP
+    trap subscription::unregister INT
+    trap subscription::unregister TERM
+    trap subscription::unregister USR1
+    trap subscription::unregister USR2
+}
+
+subscription-manager release --set=8.6
+subscription-manager refresh
+subscription::defer_unregister
+subscription-manager repos --enable rhel-8-for-x86_64-baseos-eus-rpms
+subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
+subscription-manager repos --enable rhel-8-for-x86_64-appstream-rpms
+yum -y install gettext yum-utils createrepo dnf-utils modulemd-tools
+yum clean all
+TMP_DIR="$(mktemp -d repodata-XXXX)"
+cp packages.txt "${TMP_DIR}"
+pushd "${TMP_DIR}"
+#shellcheck disable=SC2046,SC2062,SC2063,SC2035
+repoquery --archlist=x86_64,noarch --resolve --requires --recursive  $(cat packages.txt) | grep -v *.i686  >> reqs.txt
+sed -i 1d reqs.txt # we need to get rid of the first line
+#shellcheck disable=SC2046
+yumdownloader --archlist=x86_64,noarch --setopt=skip_missing_names_on_install=False -x \*i686 $(< reqs.txt)
+#shellcheck disable=SC2046
+yumdownloader --setopt=skip_missing_names_on_install=False -x \*i686 --archlist=x86_64,noarch --resolve --disablerepo=*  --enablerepo=kubernetes,rhel-8-for-x86_64-baseos-eus-rpms,codeready-builder-for-rhel-8-x86_64-rpms,rhel-8-for-x86_64-appstream-rpms $(< packages.txt)
```

Comment:
Why do we need `--enablerepo=` here but not for RHEL 8.4?

Reply:
It looks like it is in rhel 8.4

```
yumdownloader --setopt=skip_missing_names_on_install=False -x \*i686 --archlist=x86_64,noarch --resolve --disablerepo=* --enablerepo=kubernetes,rhel-8-for-x86_64-baseos-eus-rpms,codeready-builder-for-rhel-8-x86_64-rpms,rhel-8-for-x86_64-appstream-rpms --disablerepo=appstream-centos $(< packages.txt)
```

Reply:
Ahh yep you're right, I was looking at https://github.com/mesosphere/konvoy-image-builder/pull/1016/files#diff-393d0b59de96c317795575cf11a808fd2e7e0600633727daf2fd4f4f78dd8bb0R55